tests/tickets/16820/manual/ie10hrline.html

```diff
@@ -0,0 +1,14 @@
+<div id="editor">
```

Comment:
Since this tests are already in dedicated folder `tests/16820/` there is no need for such specific naming. You may simply name files `1.html`, `1.md` (unless the issue covers more cases which should be clearly distinguishable).